tests/ffi/lib.rs

```diff
@@ -342,6 +342,13 @@ pub mod ffi {
 
     impl Box<Shared> {}
     impl CxxVector<SharedString> {}
+
+    extern "Rust" {
+        type RustTypeBindingReusedAcrossModules = crate::module::RustTypeBindingReusedAcrossModules;
```

Comment:
This feels a bit wordy to me; I wonder if something like `CrossModuleRustType` would work (admittedly it's not much shorter though)